PlaybackLab/TVOSSwiftSampleApp/TVOSSwiftSampleApp/List/VideoTableViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  TVOSSwiftSampleApp
+//
+//  Copyright © 2018 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class VideoTableViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+  
+  var tableOptions : Array<PlayerSelectionOption>!
+  var tableView: UITableView!
+  
+  var rowHeight: CGFloat!
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    tableView = UITableView(frame: CGRect(x: 0, y: 0, width: self.view.frame.width, height: self.view.frame.height), style: .plain)
+    self.view.addSubview(tableView)
+    
+    self.tableView.dataSource = self
+    self.tableView.delegate = self
+    
+    self.rowHeight = 100
+    self.populateOptions()
+    
+  }
+  override func viewWillLayoutSubviews() {
+    super.viewWillLayoutSubviews()
+    tableView.frame = view.bounds
+  }
+  
+  func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+    return 1
+  }
+  
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = UITableViewCell(style: .default, reuseIdentifier: nil)
+    let option = self.tableOptions[indexPath.row]
+    cell.textLabel?.text = option.title
+    
+    return cell
+  }
+  
+  func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+    return rowHeight
+  }
+  func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+    return (rowHeight) * 0.2
+  }
+  func numberOfSections(in tableView: UITableView) -> Int {
+    return 1
+  }
+  
+// MARK: - UITableViewDataSource's methods
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return self.tableOptions.count
+  }
+  
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath){
+    let option = self.tableOptions[indexPath.row]
+    self.performSegue(withIdentifier: option.segueName, sender: self.tableView.cellForRow(at: indexPath))
+  }
+  
+  override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+    let indexPath = self.tableView.indexPathForSelectedRow
+    if (segue.identifier == "fullscreenSegue") {
```

Comment:
We should use best practices from Swift, so no `if-else if-if...` constructions should be used
```suggestion
    switch segue.identifier {
    case "fullscreensegue": ...
    case "pulsesegues: ...
    ...
```

Reply:
Done